src/go/rpk/pkg/cli/registry/schema/create.go

```diff
@@ -34,6 +37,22 @@ This uploads a schema to the registry, creating the schema if it does not
 exist. The schema type is detected by the filename extension: ".avro" or ".avsc"
 for Avro and ".proto" for Protobuf. You can manually specify the type with the 
 --type flag.
+
+You may pass the references using the --reference flag, which accepts either a
+comma separated list of <name>:<subject>:<version> or a path to a file. The file 
+must contain lines of name|subject|version separated by a tab or space, or the
```

Comment:
`|` reads as if this must be formatted with pipe -- what do you think about "name, subject, and version"?